From bfc38df65896e34c6d7d995a044ce75974757436 Mon Sep 17 00:00:00 2001
From: Victor Stinner <vstinner@python.org>
Date: Tue, 21 Jun 2022 10:27:59 +0200
Subject: [PATCH 6/8] gh-93839: Move Lib/unttest/test/ to
 Lib/test/test_unittest/ (#94043)

* Move Lib/unittest/test/ to Lib/test/test_unittest/
* Remove Lib/test/test_unittest.py
* Replace unittest.test with test.test_unittest
* Remove unittest.load_tests()
* Rewrite unittest __init__.py and __main__.py
* Update build system, CODEOWNERS, and wasm_assets.py
---
 Tools/wasm/wasm_assets.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Tools/wasm/wasm_assets.py b/Tools/wasm/wasm_assets.py
index aa09d2dda8..30aad35760 100755
--- a/Tools/wasm/wasm_assets.py
+++ b/Tools/wasm/wasm_assets.py
@@ -113,7 +113,6 @@
 # regression test sub directories
 OMIT_SUBDIRS = (
     "tkinter/test/",
-    "unittest/test/",
 )
 
 SYSCONFIG_NAMES = (
-- 
2.25.1

